src/panels/lovelace/special-rows/hui-select-row.ts

```diff
@@ -0,0 +1,84 @@
+import {
```

Comment:
Move this from `special-rows` into a new folder that contains all unused entities components.